### DISCUSSION TITLE
invalid configuration - extra keys not allowed

### DISCUSSION THREAD

**Discussion by gastonschabas** [Mergifyio/mergify#4031] [Q&A]
Hi there. I started to get the following error message in any of my pull requests created by `mergify`

```
extra keys not allowed @ pull_request_rules → item 1 → actions → merge → strict
```

The message appears in any of the following PRs: [Update sbt to 1.6.2 #184](https://github.com/gastonschabas/rumble-on-scala/pull/184), [Update postgresql to 42.3.2 #183](https://github.com/gastonschabas/rumble-on-scala/pull/183), [Update scalafmt-core to 3.3.3 #182](https://github.com/gastonschabas/rumble-on-scala/pull/182) and some [others](https://github.com/gastonschabas/rumble-on-scala/pulls) also opened by mergify.
In the past many PRs where merged using this config. Looks like the `key strict` was removed from the `action merge`. So I created [this new PR](https://github.com/gastonschabas/rumble-on-scala/pull/186/checks?check_run_id=5083817010) to remove that key, but I check the config file in the config editor and it says that is valid, but I'm still getting the same error message. Any thoughts? Thanks in advance

**Comment by jd** [comment]
Hi @gastonschabas 

Your check link points to the **current configuration error**.
Your new configuration is ok see https://github.com/gastonschabas/rumble-on-scala/pull/186/checks?check_run_id=5083818088

**Reply by gastonschabas** [reply]
Hi @jd. Thanks for your reply. Yeah, that was what I thought. Should I merge my PR https://github.com/gastonschabas/rumble-on-scala/pull/186 beside this check https://github.com/gastonschabas/rumble-on-scala/pull/186/checks?check_run_id=5083817010 is failing? Or is there a way to run the check using the latest configuration?

**Reply by jd** [reply]
Yes, you should merge it. The check failing is failing because the *current configuration in the repo* is wrong. Not the one in the PR. :)

**Reply by gastonschabas** [reply]
Oh, got it. Thanks. I merged and now everything looks like it's working. My question now is why the configuration in the repo is being used instead of the new one. If I create a new PR with a new configuration, is because I want ot use that one instead of the existing one. Thanks
